src/identifiers.rs

```diff
@@ -218,16 +218,10 @@ impl RoomId {
         Ok(Self::from(ruma::RoomId::parse(id)?))
     }
 
-    /// Returns the user's localpart.
-    #[wasm_bindgen(getter)]
-    pub fn localpart(&self) -> String {
-        self.inner.localpart().to_owned()
-    }
-
     /// Returns the server name of the room ID.
```

Comment:
this is no longer accurate.

Please please ensure that method behaviour is correctly documented. Failure to do so is one of the reasons matrix-js-sdk is such a nightmare to work on, and letting this library go down the same path would be terrible.